pkg/pmem-csi-driver/registryserver.go

```diff
@@ -16,8 +16,9 @@ type NodeInfo struct {
 	NodeID string
 	//Endpoint node controller endpoint
 	Endpoint string
-	//Capacity node capacity(unused)
-	Capacity uint64
+	//Capacity node capacity
+	CapacityFsdax  uint64
+	CapacitySector uint64
```

Comment:
@olve, i would prefer capacity as `map{string}uint64`, so that we can expand this api easily and even at controller side we can check something like `if capacity[vol.NsMode] > asked`